libs/shared/src/consts/providers/provider.interface.ts

```diff
@@ -14,6 +14,7 @@ export interface IConfigCredentials {
   key: CredentialsKeyEnum;
   value?: string;
   displayName: string;
+  credentialDescription?: string;
```

Comment:
```suggestion
  description?: string;
```

A really small suggestion, feel free to ignore. But because it's already in the `ICredential` interface, it feels like the c redential part can be omitted and replaced with just `description` for a more concise interface prop.

Reply:
good point :)